### PR TITLE
Version 6.1.0

### PR DESCRIPTION
**Pull request by samtstern**
Releasing version `6.1.0`.

Holding this PR until all relevant console changes have launched.

### CODE REVIEW

File: app/src/main/java/com/firebase/uidemo/auth/AuthUiActivity.java [hunk 1]
```diff
@@ -182,6 +185,8 @@ public void onCheckedChanged(CompoundButton compoundButton, boolean checked) {
                     mUseEmailLinkProvider.setChecked(false);
                     mUsePhoneProvider.setChecked(false);
                     mUseAnonymousProvider.setChecked(false);
+                    mUseYahooProvider.setChecked(false);
+                    mUseAppleProvider.setChecked(false);
```
Comment (SUPERCILEX) [inline app/src/main/java/com/firebase/uidemo/auth/AuthUiActivity.java]: Should `mUseMicrosoftProvider` also be here?

File: app/src/main/java/com/firebase/uidemo/util/ConfigurationUtils.java [hunk 2]
```diff
@@ -67,7 +67,8 @@ public static boolean isTwitterMisconfigured(@NonNull Context context) {
 
 
         providers.add(new AuthUI.IdpConfig.PhoneBuilder().build());
-
+        providers.add(new AuthUI.IdpConfig.MicrosoftBuilder().build());
+        providers.add(new AuthUI.IdpConfig.YahooBuilder().build());
```
Comment (SUPERCILEX) [inline app/src/main/java/com/firebase/uidemo/util/ConfigurationUtils.java]: And `AppleBuilder` here?

File: auth/build.gradle.kts [hunk 3]
```diff
@@ -13,6 +13,10 @@ android {
         disable("UnknownNullness")  // TODO fix in future PR
         disable("TypographyQuotes") // Straight versus directional quotes
         disable("DuplicateStrings")
+        disable("MissingTranslation") // TODO add translations
```
Comment (SUPERCILEX) [inline auth/build.gradle.kts]: Have these gone in?

File: README.md [hunk 4]
```diff
@@ -48,16 +48,16 @@ libraries.
 ```groovy
 dependencies {
     // FirebaseUI for Firebase Realtime Database
-    implementation 'com.firebaseui:firebase-ui-database:6.0.2'
+    implementation 'com.firebaseui:firebase-ui-database:6.0.3'
```
Comment (SUPERCILEX) [inline README.md]: This should be `6.1.0`, right?

**Comment by samtstern** [comment]
@SUPERCILEX thanks for catching those!  As for #1714 I'd like to launch Generic IDP for new providers before using it to replace anything existing.  That way if we have any big issues they aren't regressions.

I will fix those issues on `6.0.3-dev` and rebase.

**Comment by SUPERCILEX** [comment]
@samtstern SGTM all around 🎉

File: auth/build.gradle.kts (hunk 3 above)
Comment (samtstern) [reply auth/build.gradle.kts]: Looks like they're ready now, I'll go get them

**Pull request merged**